dts/riscv/gigadevice/gd32vf103.dtsi

```diff
@@ -40,7 +40,7 @@
 		systimer: timer@d1000000 {
 			compatible = "nuclei,systimer";
 			reg = <0xd1000000 0x10000>;
-			interrupts = <3 0>, <7 0>;
```

Comment:
you should fix drivers/timer/riscv_machine_timer.c by using `DT_INST_IRQ_BY_IDX(0, 1, irq)` instead.

Reply:
(and btw, sorry to introduce this issue)